plugins/scaffolder/src/next/types.ts

```diff
@@ -15,6 +15,25 @@
  */
 import type { FormProps as SchemaFormProps } from '@rjsf/core-v5';
 
+/**
+ * The field template from \@rjsf/core which is a react component that gets passed \@rjsf/core field related props.
```

Comment:
Think you can backticks instead of escaping here to remove the warning and make it look nice in the docs.

Reply:
done